app/src/main/kotlin/com/simplemobiletools/smsmessenger/receivers/MmsReceiver.kt

```diff
@@ -50,5 +51,5 @@ class MmsReceiver : MmsReceivedReceiver() {
         }
     }
 
-    override fun onError(context: Context, error: String) {}
+    override fun onError(context: Context, error: String) = context.toast(R.string.couldnt_download_mms)
```

Comment:
lets show a long toast at errors, or just reuse the helper showErrorToast() context extension